packages/sveltekit-plugin/src/sveltekit-bundle-analysis/getBundleName.ts

```diff
@@ -0,0 +1,22 @@
+export function getBundleName(
+  initialName = "",
+  initialDir = "",
+  format: string,
```

Comment:
nit: this can be typed as `ModuleFormat` from rollup. Same as other places that use this.

Reply:
Would it be alright to re-export this type from the Vite bundler plugin package?

Reply:
yeah that's fine!

Reply:
Hrrm it actually doesn't seem like Vite re-exports these types, soooo I'd have to install Rollup as a dev dep eh?

Reply:
Can you grab the type from `UnpluginOptions`? Otherwise don't worry too much, we can leave it as is.

Reply:
Checked the options ... and no dice with that.